challenge_1/c/xoumi/ReverseString.c

```diff
@@ -0,0 +1,27 @@
+#include<stdio.h>
+void main()
+{
+    int i,j;
+    char *string,temp;
+    printf("Enter string: ");
+    //because scanf is scary
+    gets(string);
+
+    //Just a loop to count length instead of using strlen because this isn't scary
+    for(i=0;string[i]!='\0';i++);
```

Comment:
Is there a reason not to use `strlen` I don't know of any reason it would be considered scary.